.storybook-sdk/preview.tsx

```diff
@@ -42,3 +44,16 @@ export const globalTypes: GlobalTypes = {
     },
   },
 };
+
+/*
+ * Initializes MSW
+ * See https://github.com/mswjs/msw-storybook-addon#configuring-msw
+ * to learn how to customize it
+ */
+
+initialize({
+  onUnhandledRequest: "bypass",
+});
+const preview = { parameters, decorators, globalTypes, loaders: [mswLoader] };
```

Comment:
Refer to the [installation instruction here](https://storybook.js.org/docs/writing-stories/mocking-data-and-modules/mocking-network-requests). I name the directory like this so it's clear, that's only used for msw.